frontend/src/components/messages/NewRequestDialog.vue

```diff
@@ -173,6 +332,46 @@ export default {
     someFacilitiesSelected() {
       return this.newRequestModel.facilities.length > 0
     },
+    isAnAccountMaintenanceRequest() {
+      return this.newRequestModel.requestCategoryId === REQUEST_CATEGORY_TYPES.ACCOUNT_MAINTENANCE
+    },
+    isAnySubCategoryChecked() {
+      return this.newRequestModel.subCategories.length > 0
+    },
+    isOrganizationDetailsChecked() {
+      return this.newRequestModel.subCategories.some(subCategory => subCategory.subCategoryId === REQUEST_SUB_CATEGORY_TYPES.ORGANIZATION_DETAILS)
+    },
+    isFacilityDetailsChecked() {
+      return this.newRequestModel.subCategories.some(subCategory => subCategory.subCategoryId === REQUEST_SUB_CATEGORY_TYPES.FACILITY_DETAILS)
+    },
+    isOrgPhoneEmailChecked() {
```

Comment:
These functions seem pretty repetitive. Would a function like this be suitable? Where category is the relevant constant variable

isSubCatChecked(category) {
      return this.newRequestModel.subCategories.some((subCategory) => subCategory.subCategoryId === category)
    },